### PR TITLE
Petra as in G&K fix

### PR DESCRIPTION
**Pull request by ravignir**
x

### CODE REVIEW

**Comment by yairm210** [comment]
Change 'providesFreeBuilding' field to "Provides a free [Ampitheater] [in this city]" unique - new and improved version :)

**Comment by SomeTroglodyte** [comment]
The + - game still irks. A direct solution would be to code a "desert but not flood plains" specialized tileFilter... Which raises the question: Our Petra as defines should affect Desert+Hill and Desert+Oasis - does the original?

**Comment by ravignir** [comment]
> 
> The + - game still irks. A direct solution would be to code a "desert but not flood plains" specialized tileFilter... Which raises the question: Our Petra as defines should affect Desert+Hill and Desert+Oasis - does the original?

x

**Comment by ravignir** [comment]
> 
> 
> Change 'providesFreeBuilding' field to "Provides a free [Ampitheater] [in this city]" unique - new and improved version :)

done

**Comment by logicminimal** [comment]
> The + - game still irks. A direct solution would be to code a "desert but not flood plains" specialized tileFilter... Which raises the question: Our Petra as defines should affect Desert+Hill and Desert+Oasis - does the original?

i think terrain combinations are good enough, and if nothing is inputted per one tile it should ignore that empty one

like plains + hills +forest, this is obvious but if it doesnt have forest it should apply for any that containts at least those two

File: android/assets/jsons/Civ V - Vanilla/Buildings.json [hunk 1]
```diff
@@ -390,7 +390,7 @@
 		"culture": 1,
 		"isWonder": true,
 		"greatPersonPoints": {"Great Engineer": 1},
-		"uniques": ["Must be next to [Desert]", "[+1 Food, +1 Production] from [Desert] tiles [in this city]", "[-1 Food, -1 Production] from [Flood plains] tiles [in this city]", "[+6 Culture] once [Archaeology] is discovered"],
+		"uniques": ["Must be next to [Desert]", "[+1 Food, +1 Production, +1 Gold] from [Desert] tiles [in this city]", "[-1 Food, -1 Production, -1 Gold] from [Flood plains] tiles [in this city]", "Provides a free [Amphitheater] [in this city]"], "[+6 Culture] once [Archaeology] is discovered"],
```
Comment (yairm210) [inline android/assets/jsons/Civ V - Vanilla/Buildings.json:393]: There's an extra ] at the end of "in this city" that's causing the test errors

Comment (ravignir) [reply android/assets/jsons/Civ V - Vanilla/Buildings.json]: ok

**Pull request merged**